utils/rpms/packaging/rpm_chrootbuild

```diff
@@ -21,6 +21,12 @@ ln -sf /etc/mock/logging.ini "$mock_config_dir/"
 
 cp "$original_cfg_file" "$cfg_file"
 
+if [ "${CHROOT_NAME}" == "epel-8-x86_64" ]; then
+       >> "$cfg_file"
```

Comment:
(lint) This redirection doesn't have a command. Move to its command (or use 'true' as no-op). [SC2188]